osquery/tables/networking/darwin/wifi_utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreWLAN/CoreWLAN.h>
+#include <iomanip>
+// SSIDs have no character set associated with them
+// mirror Apple's representation of them
+std::string extractSsid(const CFDataRef& data);
+
+// Change an Apple Constant into a human readable string representing
+// the network encryption type
+std::string getSecurityName(CWSecurity cw);
```

Comment:
What type is this input argument? Can you improve performance by passing a const reference?